File: Java/프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.java
import java.util.stream.IntStream;
class Solution {
    public String solution(int[] food) {
        String answer = "";
        
        for(int i=1;i<food.length;i++){
             for(int j=0; j<food[i]/2; j++){
                 answer+=i+"";
             }
        }
        StringBuffer sb = new StringBuffer(answer);
        String rb = sb.reverse().toString();
        return answer+"0"+rb;
    }
}

File: Kotlin/프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        var answer = StringBuilder()
        for(i in 1..food.lastIndex) {
            if(food[i]/2 > 0) {
                for(j in 1..food[i]/2 ) {
                    answer.append(i)
                }
            }
        }
        return answer.toString()+"0"+answer.toString().reversed()
    }
}